docs/book/src/cronjob-tutorial/testdata/project/api/v1/cronjob_types.go

```diff
@@ -48,12 +48,11 @@ import (
 
  - A deadline for starting jobs (if we miss this deadline, we'll just wait till
    the next scheduled time)
- - What to do if multiple jobs would run at once (do we wait? stop the old one? run both?)
+ - What to do if multiple jobs would run at once (do we wait? stop the old one? or run both?)
  - A way to pause the running of a CronJob, in case something's wrong with it
  - Limits on old job history
 
- Remember, since we never read our own status, we need to have some other way to
- keep track of whether a job has run.  We can use at least one old job to do
+ Remember, since we never read our status, we need to have another way to track whether a job has run.  We can use at least one old job to do
```

Comment:
Will not it fail in the lint?
The line should still broke as before. 
Also, why remove own?